cmd/mattermost/commands/config.go

```diff
@@ -320,10 +320,27 @@ func UpdateMap(configMap map[string]interface{}, configSettings []string, newVal
 		if len(configSettings) == 1 {
 			return errors.New("unable to set multiple settings at once")
 		}
-		if value.Len() == 0 {
-			return fmt.Errorf(noSettingsNamed, configSettings[1])
+		simpleMap, ok := res.(map[string]interface{})
+		if ok {
+			return UpdateMap(simpleMap, configSettings[1:], newVal)
 		}
-		return UpdateMap(res.(map[string]interface{}), configSettings[1:], newVal)
+		mapOfTheMap, ok := res.(map[string]map[string]interface{})
```

Comment:
It took me a bit to wrap my head around why the type assertion on line 323 doesn't match here. Intuitively, I think I had thought of `interface{}` in this kind of context as a glob of sorts, but need to remember that it's a concrete type itself like any other. Thanks for the reminder :)